manifests/02-namespace.yaml

```diff
@@ -4,10 +4,15 @@ metadata:
   name: openshift-console
   annotations:
     openshift.io/node-selector: ""
+  # console does not yet expose metrics, but once it does, we can turn this on
+  # labels:
+  #   openshift.io/cluster-monitoring: "true"
```

Comment:
I'd prefer to remove the commented out code here as well. This is trivial to add later when we need it.

Reply:
can do.